index.html

```diff
@@ -0,0 +1,738 @@
+                                   </div>
+                                 </a>
+                                 </div>
+                
+                <ul>
+                 <li><a href="#"><i class="fas fa-tag"></i>Messaging, +4</a></li>
+                 <li><a href="#"><i class="fas fa-briefcase"></i>Standard Chartered Bank</a></li>
+                </ul>
+                <p><a href="#">US-based collaboration platform Symphony bags $165m to expand into Asia <span class="index">on Tech in Asia</span></a></p>
+                </div>
+            
+                <div class="buy">
+                        <div class="fund-group">
+                                <a href="#">
+                                   <img src="images/buyandship.jpg" alt="buy and ship">
+                                   <h4 class="fund-group-name">Buyandship</h4>
+                                   <div class="fund-detail">
+                                       <h4>$2.2M</h4>
+                                       <h5>SERIES A</h5>
+                                   </div>
+                                 </a>
+                                 </div>
+                        
+                        <ul>
+                         <li><a href="#"><i class="fas fa-tag"></i>E-commerce, +3</a></li>
+                         <li><a href="#"><i class="fas fa-briefcase"></i>Infinity Venture Capital</a></li>
+                        </ul>
+                        <p><a href="#">Ecommerce shipping startup banks $2.2m in pre-series B round <span class="index">on Tech in Asia</span></a></p>
+                        </div>
+                
+                </div>
+
+
+
+        <div class="group1">
+            <div class="apps group-item">
+            <h3 class="category-title"><a href="#">Apps <i class="fas fa-greater-than"></i></a></h3>
+              <div class="apps-image">
+                  <div class="overlay3"></div>
+                  <img src="images/apps.png" alt="mind mapping">
+                  <div class="group-item-text">
+                  <span class="image-title">MOST POPULAR</span>
+                  <h4 class="article-title"><a href="#">Report: Google Maps hosts 'millions' of fake business listings</a></h4>
+                  <span class="article-author-detail"><a href="#">Matthew Beedham</a>7 hours ago</span>
+              </div>
+            </div>
+
+              <div class="related">
+
+                  <div class="related-details">
+                      <h5 class="related-latest">LATEST</h5><br>
+                      <span class="related-title"><a href="#">Google's new extension will help you report dubious sites with a click</a></span>
+
+                  </div>
+                  <img src="images/latest1.png" class="related-image" alt="">
+              </div>
+               <ul class="group-list">
+                   <li><a href="#">YouTube will soon let you try out makeup with AR</a></li>
```

Comment:
The anchor tag must be inside an `article` tag. Kindly do that with all list elements.